sdk/typescript/test/e2e/signature-mpc.test.ts

```diff
@@ -103,12 +103,19 @@ describe('Test signature mpc', () => {
 			encryptionKeyObj.objectID,
 		);
 		const message: Uint8Array = new TextEncoder().encode('Sign it!!!');
-		let signatures = await signWithDWalletID(
+		let presignObjID = await presignWithDWalletID(
 			toolbox.client,
 			toolbox.keypair,
 			dwallet?.dwalletId!,
 			message,
 		);
+		let signatures = await approveAndSign(
+			dwallet?.dwalletCapId!,
+			presignObjID!,
+			[message],
+			toolbox.keypair,
+			toolbox.client,
+		);
 		console.log({ signatures });
```

Comment:
Let's add here:
- fetch the encrypted user share of the dwallet, verify signature over the public keys, and get the public key.
- verify the signatures returned from `approveAndSign` against the public key. If it fails, don't return the signatures, return an error - so that callers don't ever get wrong signatures.